crates/spl_network_messages/src/lib.rs

```diff
@@ -18,17 +18,34 @@ pub use game_controller_state_message::{
     GameControllerStateMessage, GamePhase, GameState, Half, Penalty, PenaltyShoot, Player,
     SubState, Team, TeamColor, TeamState,
 };
-pub use visual_referee_message::{VisualRefereeDecision, VisualRefereeMessage};
+pub use visual_referee_message::{GestureVisualRefereeDecision, GestureVisualRefereeMessage};
+
+#[derive(Clone, Copy, Debug, Deserialize, Serialize)]
+pub enum HulkMessage {
+    Striker(StrikerMessage),
+    VisualReferee(VisualRefereeMessage),
+}
+
+impl Default for HulkMessage {
+    fn default() -> Self {
+        HulkMessage::Striker(StrikerMessage::default())
+    }
+}
```

Comment:
Why do we need `Default`?

Reply:
Hmm, `IncomingMessage` and `Outgoing` message are outputs and therefore require `Default`. I could remove the default implementations of `HulkMessage`, ... but then there would need to be some hardcoded default in the `Default` implementation of `Incoming/OutgoingMessage`

Reply:
This may have to be improved in some next iteration of the framework. There are several `Default`s flying around that should not be there. Ok for now, but we should try to not implement traits for types that are not designed to implement these...

resolving for now